eng/pipelines/tools-repo-versioning.yml

```diff
@@ -36,4 +36,4 @@ jobs:
           - azure-sdk-for-java
           - azure-sdk-for-js
           - azure-sdk-for-net
-#          - azure-sdk-for-python
\ No newline at end of file
+          - azure-sdk-for-python
```

Comment:
Can we not easily remove python from the list of repo's we need to update? Once you are done with the overall epic I'm hopefully that we will only need java and net in this list as they still include signing stuff from our internal tools repo but everything else I believe should be coming from the eng/common directory.

Reply:
I can remove this since they js and python repos no longer use repository resources for any of the tool repos. However I was wondering if we still want to have the ref updated connected in the case that someone add a repository resource at some other time.

Reply:
That is fair I'd be OK keeping these in here if the result is a no-op for those repos that don't have references.